db/oracle_schema.rb

```diff
@@ -96,19 +96,19 @@
     t.datetime "proxy_configs_conf_updated_at"
     t.datetime "hosted_proxy_deployed_at"
     t.string   "po_number"
-    t.datetime "deleted_at"
+    t.datetime "state_changed_at"
   end
 
   add_index "accounts", ["default_service_id"], name: "index_accounts_on_default_service_id"
-  add_index "accounts", ["domain", "deleted_at"], name: "index_accounts_on_domain_and_deleted_at"
+  add_index "accounts", ["domain", "state_changed_at"], name: "index_accounts_on_domain_and_state_changed_at"
```

Comment:
Changing the index might block the DB. Verification needed.

Reply:
> Changing the index might block the DB
Yup, so for Saas we will do the migration separately 😄 But this is Oracle so it is in on-prem, so what can we do? and what do you mean by `veritication needed`? 😕